### PR TITLE
Fix failing test in Plone 4.3. Add Plone 5.0/5.1 Buildout. Add Travis configuration.

### PR DESCRIPTION
**Pull request by tisto**

### CODE REVIEW

File: src/plone/app/workflowmanager/browser.txt [hunk 1]
```diff
@@ -34,7 +34,7 @@ There should not be any plone workflows by default listed.
     False
 
     >>> 'one_state_workflow' in browser.contents
-    False
+    True
```
Comment (jaroel) [inline src/plone/app/workflowmanager/browser.txt]: This doesn't look right, given "There should not be any plone workflows by default listed", especially when allow_failures is on for p5.0 and p5.1

Comment (tisto) [reply src/plone/app/workflowmanager/browser.txt]: @jaroel the plone.app.contenttypes test layer enables the one_state_workflow by default. If I recall correctly this was even added in a minor release of p.a.contenttypes.

Comment (tisto) [reply src/plone/app/workflowmanager/browser.txt]: @jaroel here you go: https://github.com/plone/plone.app.contenttypes/blob/1.1.x/plone/app/contenttypes/testing.py#L64

Comment (jaroel) [reply src/plone/app/workflowmanager/browser.txt]: Thats a different workflow :D

Comment (tisto) [reply src/plone/app/workflowmanager/browser.txt]: Wait. this is simple_publication_workflow, we have one_state_workflow. I might have been on the wrong track...

Comment (tisto) [reply src/plone/app/workflowmanager/browser.txt]: @jaroel I digged a bit deeper and it turns out this is actually the comment_one_state_workflow which bleeds into the test. I have no idea why plone.app.discussion is enabled in the default test layer here. That shouldn't be the case. I couldn't find any traces of it in plone.app.testing thg.

Comment (tisto) [reply src/plone/app/workflowmanager/browser.txt]: @jaroel turned out to be plone.app.contenttypes after all: https://github.com/plone/plone.app.contenttypes/blob/1.1.x/plone/app/contenttypes/profiles/default/metadata.xml#L11

**Comment by tisto** [comment]
I agree with @jaroel that we have to investigate the workflow issue. Though, this clearly is an issue of the test layer, not of the p.a.workflowmanager package. See: 

https://github.com/plone/plone.app.contenttypes/issues/432

I added a note to the test. Though, I would suggest to merge this pr because we desperately need a working testing/ci for this package.

**Comment by jaroel** [comment]
Agree, we'll fix this later on.

**Pull request merged**